packages/kit/src/runtime/client/client.js

```diff
@@ -663,7 +663,7 @@ export function create_client(app, target) {
 			// if `paths.base === '/a/b/c`, then the root route is `/a/b/c/`,
 			// regardless of the `trailingSlash` route option
```

Comment:
```suggestion
			// if `paths.base === '/a/b/c`, then the root route is always `/a/b/c/`, regardless of
			// the `trailingSlash` route option, so that relative paths to JS and CSS work
```